apis/Google.Cloud.Vision.V1/Google.Cloud.Vision.V1.Tests/ImageAnnotatorClientTest.cs

```diff
@@ -174,20 +174,13 @@ public class ImageAnnotatorClientTest
             AssertAnnotationsSuccess(allAnnotations, await asyncCall(s_allAnnotationsImage, null, 0, null));
             AssertAnnotationsSuccess(allAnnotations.Take(1), syncCall(s_allAnnotationsImage, null, 1, null));
             AssertAnnotationsSuccess(allAnnotations.Take(1), await asyncCall(s_allAnnotationsImage, null, 1, null));
-            AssertAnnotationError(syncCall(s_errorImage, null, 0, null));
-            AssertAnnotationError(await asyncCall(s_errorImage, null, 0, null));
+            Assert.Throws<AnnotateImageException>(() => syncCall(s_errorImage, null, 0, null));
+            await Assert.ThrowsAsync<AnnotateImageException>(() => asyncCall(s_errorImage, null, 0, null));
         }
 
-        private void AssertAnnotationsSuccess<T>(IEnumerable<T> expectedAnnotations, AnnotationResult<T> actualResult)
+        private void AssertAnnotationsSuccess<T>(IEnumerable<T> expectedAnnotations, IReadOnlyList<T> actualResult)
```

Comment:
Uh, yes. I meant to remove it having checked that the previous ToList calls weren't needed. Will fix.